### PR TITLE
Setup testing environment

### PR DESCRIPTION
**Pull request by Mavrin**
I created setup for tests and write few tests on component and model.

### CODE REVIEW

File: web/app/common/store.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import store from './store';
+
+describe('Store', () => {
+  it('should trigger listener on add comment', () => {
+    const listener = jest.fn();
```
Comment (igoradamenko) [inline web/app/common/store.test.js]: Can you add some space between lines, please? Smth like this:

```
const listener = jest.fn();

store.set('comments', []);
store.onUpdate('comments', listener);
store.addComment({id:`new`});

expect(listener).toBeCalledWith([{"comment": {"id": "new"}}]);
```

And so on in the others files.

File: web/app/components/auth-panel/auth-panel.jsx [hunk 2]
```diff
@@ -50,7 +51,7 @@ export default class AuthPanel extends Component {
         }
 
         {
-          !user.id && (
+          loggedIn === false && (
```
Comment (igoradamenko) [inline web/app/components/auth-panel/auth-panel.jsx]: `!loggedIn`?

Comment (Mavrin) [reply web/app/components/auth-panel/auth-panel.jsx]: Fixed. But  I don't like `expression && doSomething()` pattern. It is not easy read.

Comment (igoradamenko) [reply web/app/components/auth-panel/auth-panel.jsx]: I understand, but it's the only way to use conditions inside JSX ¯\_(ツ)_/¯.

Comment (Mavrin) [reply web/app/components/auth-panel/auth-panel.jsx]: I prefer to use  isLogged?<UserInfo>: <AuthForm>

File: web/app/common/store.test.js (hunk 1 above)
Comment (Mavrin) [reply web/app/common/store.test.js]: Fixed

File: web/app/components/auth-panel/auth-panel.jsx (hunk 2 above)
Comment (igoradamenko) [reply web/app/components/auth-panel/auth-panel.jsx]: Oh, I got it.

We have #96 for it. So if you want you can create these components and use a ternary operator here, in your next PR ^__^

**Pull request merged**